core/src/main/resources/schema/postgres/postgres-create-schema.sql

```diff
@@ -47,3 +47,16 @@ CREATE TABLE IF NOT EXISTS public.snapshot (
   PRIMARY KEY(persistence_id, sequence_number)
 );
 
+CREATE TABLE IF NOT EXISTS "state" (
+    "global_offset" BIGSERIAL,
+    "persistence_id" VARCHAR(255) NOT NULL,
+    "sequence_number" BIGINT NOT NULL,
+    "state_payload" BYTEA NOT NULL,
+    "state_serial_id" INTEGER NOT NULL,
+    "state_serial_manifest" VARCHAR(255),
+    "tag" VARCHAR,
+    "state_timestamp" BIGINT NOT NULL,
+    PRIMARY KEY("persistence_id")
+    );
+CREATE INDEX CONCURRENTLY "state_tag_idx" on "state" ("tag");
+CREATE INDEX CONCURRENTLY "state_global_offset_idx" on "state" ("global_offset");
```

Comment:
@octonato I found it to be a recommended practice in Postgres. https://www.2ndquadrant.com/en/blog/create-index-concurrently/

> A unique distinguishing factor of CIC is that it can build a new index on the table, without blocking it from updates/inserts/deletes.